requirements/requirements.yaml

```diff
@@ -20,6 +20,7 @@ dependencies:
   - conda-build>=3.23
   - flake8>=6.0
   - flake8-black>=0.3
+  - frozendict>=2.0
```

Comment:
Are you introducing frozendict because it is hashable? If that's the case then please consider using either namedtuple or a frozen dataclass.